dsa-solutions/lc-solutions/0000-0099/0018-4Sum.md

```diff
@@ -0,0 +1,286 @@
+---
+id: 4Sum
+title: 4Sum (LeetCode)
+sidebar_label: 0018-4Sum.md
+tags:
+    - Arrays
+    - Hashing
```

Comment:
remove 

```md
 such that:

- `0 <= a, b, c, d < n`
- `a, b, c, and d are distinct`
- `nums[a] + nums[b] + nums[c] + nums[d] == target`
```

because this is not good practice